Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,9 @@
 import React from 'react';
 import './Button.css';
+
+
+const ActionButton = props => {
+    return (<button className={props.buttonStyle}>{props.text}</button>);
```

Comment:
```suggestion
    return <button className={props.buttonStyle}>{props.text}</button>;
```
When you are justing returning one thing you don't need '( )' and you can go even further and remove the '{ }' make it go on one line without the return keyword